connect/src/com/telenor/connect/id/Claims.java

```diff
@@ -8,6 +8,7 @@
 
     public static final String OPENID = "openid";
     public static final String PROFILE = "profile";
```

Comment:
In addition, it would have been nice to make the SDK doc about claims clearer. The claims set end up as so-called essential claims. It should be made clear what the client achieves by doing this. You should also evaluate whether you should support non-essential claims, which overlaps a bit with scope values.